atom/common/platform_util_win.cc

```diff
@@ -231,63 +232,65 @@ HRESULT DeleteFileProgressSink::ResumeTimer() {
 
 namespace platform_util {
 
-bool ShowItemInFolder(const base::FilePath& full_path) {
+void ShowItemInFolderOnWorkerThread(const base::FilePath& full_path) {
```

Comment:
Move this helper function to an anonymous namespace

Reply:
Thank you guys! Also want to thank God for this financial blessing. Willie Parish